integration-cli/docker_cli_events_unix_test.go

```diff
@@ -22,35 +23,25 @@ func (s *DockerSuite) TestEventsRedirectStdout(c *check.C) {
 	dockerCmd(c, "run", "busybox", "true")
 
 	file, err := ioutil.TempFile("", "")
-	if err != nil {
-		c.Fatalf("could not create temp file: %v", err)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("could not create temp file: %v", err))
```

Comment:
Just `check.Commentf("could not create temp file")` is enough , the `checker.IsNil` will display the error if it not `nil`